src/test/resources/records/GenericRecord.java

```diff
@@ -0,0 +1,3 @@
+package records;
+
+public record GenericRecord<T>(T a, T b) {}
```

Comment:
Could you please add this test case under `src/test/resources/records`? It's where we put all the tests related to records.

Reply:
Hm... I think the suggested path is the same as where the file is currently located. But I think I should update the package name.